### ISSUE TITLE
timeout on CentOS & MacOS; no timeout on Ubuntu

### ISSUE THREAD

**Issue by janssen** [sbraconnier/jodconverter#82]
Here's a weird one.  This conversion works fine on Ubuntu 16.04, but reliably fails on a CentOS-based Docker container (my test CI) and on MacOS with the following error.  There seem to be three timeouts that this could be: processTimeout, taskExecutionTimeout, and taskQueueTimeout.  How do I tell which of the three it is?  Also, you'll note it times out after just one second.

It's just a particular document, too.  Other documents run just fine on both CentOS and MacOS.  It's a PDF to TaggedPDF conversion.

```
templatePath is /opt/app-root/src/.config/libreoffice/4/user
Jun 15, 2018 5:27:52 PM org.jodconverter.office.OfficeProcessManager submitAndWait
INFO: Submitting task 'Start' and waiting...
Jun 15, 2018 5:27:52 PM org.jodconverter.office.OfficeProcess prepareProcessBuilder
INFO: Command: /usr/lib64/libreoffice/program/soffice.bin --accept=socket,host=127.0.0.1,port=2002,tcpNoDelay=1;urp;StarOffice.ServiceManager -env:UserInstallation=file:///tmp/.jodconverter_socket_host-127.0.0.1_port-2002_tcpNoDelay-1 --invisible --norestore --nofirststartwizard --nologo --nodefault --nolockcheck --headless --nocrashreport
Jun 15, 2018 5:27:52 PM org.jodconverter.office.OfficeProcess start
INFO: Starting process with acceptString 'socket,host=127.0.0.1,port=2002,tcpNoDelay=1;urp;StarOffice.ServiceManager' and profileDir '/tmp/.jodconverter_socket_host-127.0.0.1_port-2002_tcpNoDelay-1'
Jun 15, 2018 5:27:52 PM org.jodconverter.office.OfficeProcess start
INFO: Started process; pid = 167
Jun 15, 2018 5:27:52 PM org.jodconverter.office.ConnectRetryable attempt
WARNING: Office process died with exit code 81; restarting it
Jun 15, 2018 5:27:53 PM org.jodconverter.office.OfficeProcess prepareProcessBuilder
INFO: Command: /usr/lib64/libreoffice/program/soffice.bin --accept=socket,host=127.0.0.1,port=2002,tcpNoDelay=1;urp;StarOffice.ServiceManager -env:UserInstallation=file:///tmp/.jodconverter_socket_host-127.0.0.1_port-2002_tcpNoDelay-1 --invisible --norestore --nofirststartwizard --nologo --nodefault --nolockcheck --headless --nocrashreport
Jun 15, 2018 5:27:53 PM org.jodconverter.office.OfficeProcess start
INFO: Starting process with acceptString 'socket,host=127.0.0.1,port=2002,tcpNoDelay=1;urp;StarOffice.ServiceManager' and profileDir '/tmp/.jodconverter_socket_host-127.0.0.1_port-2002_tcpNoDelay-1'
Jun 15, 2018 5:27:53 PM org.jodconverter.office.OfficeProcess start
INFO: Started process; pid = 187
Jun 15, 2018 5:27:53 PM org.jodconverter.office.OfficeConnection connect
INFO: Connected: 'socket,host=127.0.0.1,port=2002,tcpNoDelay=1'
Jun 15, 2018 5:27:53 PM org.jodconverter.task.LocalConversionTask execute
INFO: Executing local conversion task...
Jun 15, 2018 5:28:53 PM org.jodconverter.office.OfficeProcessManager restartDueToTaskTimeout
INFO: Executing task 'Restart After Timeout'...
Jun 15, 2018 5:28:53 PM org.jodconverter.office.OfficeProcess forciblyTerminate
INFO: Trying to forcibly terminate process: 'host=127.0.0.1,port=2002,tcpNoDelay=1'; pid: 187
Jun 15, 2018 5:28:53 PM org.jodconverter.office.AbstractOfficeManagerPool doStop
INFO: Stopping the office manager pool...
Jun 15, 2018 5:28:53 PM org.jodconverter.task.LocalConversionTask execute
SEVERE: Local conversion failed.
com.sun.star.uno.RuntimeException: com.sun.star.lib.uno.environments.remote.JobQueue.removeJob - unexpected:java.lang.InterruptedException
	at com.sun.star.lib.uno.environments.remote.JobQueue.removeJob(JobQueue.java:222)
	at com.sun.star.lib.uno.environments.remote.JobQueue.enter(JobQueue.java:330)
	at com.sun.star.lib.uno.environments.remote.JobQueue.enter(JobQueue.java:303)
	at com.sun.star.lib.uno.environments.remote.JavaThreadPool.enter(JavaThreadPool.java:87)
	at com.sun.star.lib.uno.bridges.java_remote.java_remote_bridge.sendRequest(java_remote_bridge.java:636)
	at com.sun.star.lib.uno.bridges.java_remote.ProxyFactory$Handler.request(ProxyFactory.java:146)
	at com.sun.star.lib.uno.bridges.java_remote.ProxyFactory$Handler.invoke(ProxyFactory.java:128)
	at com.sun.proxy.$Proxy4.loadComponentFromURL(Unknown Source)
	at org.jodconverter.task.AbstractLocalOfficeTask.loadDocument(AbstractLocalOfficeTask.java:105)
	at org.jodconverter.task.LocalConversionTask.execute(LocalConversionTask.java:102)
	at org.jodconverter.office.OfficeProcessManagerPoolEntry.doExecute(OfficeProcessManagerPoolEntry.java:148)
	at org.jodconverter.office.AbstractOfficeManagerPoolEntry$1.call(AbstractOfficeManagerPoolEntry.java:77)
	at org.jodconverter.office.AbstractOfficeManagerPoolEntry$1.call(AbstractOfficeManagerPoolEntry.java:73)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

Jun 15, 2018 5:28:53 PM org.jodconverter.office.OfficeProcessManager submitAndWait
INFO: Submitting task 'Stop' and waiting...
Jun 15, 2018 5:28:53 PM org.jodconverter.office.OfficeConnection disposing
INFO: Disconnected: 'socket,host=127.0.0.1,port=2002,tcpNoDelay=1'
Jun 15, 2018 5:28:53 PM org.jodconverter.office.OfficeProcessManager doTerminateProcess
INFO: process forcibly terminated with code 137
Jun 15, 2018 5:28:53 PM org.jodconverter.office.OfficeProcess forciblyTerminate
INFO: Trying to forcibly terminate process: 'host=127.0.0.1,port=2002,tcpNoDelay=1'; pid: 187
Jun 15, 2018 5:28:53 PM org.jodconverter.office.OfficeProcessManager doTerminateProcess
INFO: process forcibly terminated with code 137
Jun 15, 2018 5:28:53 PM org.jodconverter.office.OfficeProcessManager doEnsureProcessExited
INFO: process exited with code 137
Jun 15, 2018 5:28:53 PM org.jodconverter.office.AbstractOfficeManagerPool doStop
INFO: Office manager stopped
Required conversion step failed: org.jodconverter.office.OfficeException: Task did not complete within timeout
```

**Comment by sbraconnier** [comment]
In my personal experience, `com.sun.star.uno.RuntimeException` may occur when the input or output format is not supported by the filter being used for the conversion. For example, if I use the `draw8_template` filter to convert a docx document to odt, I'll get this exception. 

So I think you get this exception because the PDF to TaggedPDF is not supported out of the box for the combo LibreOffice/CentOS combo you are using for your CI. I also had to deal with this because I created some tests that worked on my Windows but failed when executed by TravisCI (once pushed to GitHub).

I decided to skip the conversions containing in/out format not supported when the project is built by TravisCI.
